lib/chef/policy_builder/policyfile.rb

```diff
@@ -133,6 +133,11 @@ def build_node
         # consume_external_attrs may add items to the run_list. Save the
         # expanded run_list, which we will pass to the server later to
         # determine which versions of cookbooks to use.
+
+        unless Chef::Config[:policy_document_native_api]
+          Chef.deprecated(:policyfile_compat_mode, "The chef-server 11 policyfile compat mode is deprecated, please set policy_document_native_api to true in your config")
```

Comment:
Does this get a deprecation number?

Reply:
uh yeah looks like that got lost in a rebase

Reply:
fixed